tests/unit/Infrastructure/Teams/Client/PublishEntityClientTest.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+/** Copyright 2021 SURFnet B.V.
```

Comment:
The Copyright line must be on the next line (4). That fixes your static-analysis issue (the docheader at least)

Reply:
superb, much obliged!